frontend/src/components/pages/RegistrantExperience/ReviewRegistration/index.tsx

```diff
@@ -1,26 +1,64 @@
 import React from "react";
-import { Box, Checkbox, Text } from "@chakra-ui/react";
+import { Box } from "@chakra-ui/react";
+import PaymentSummary from "./PaymentSummary";
+import { CartItem } from "../../../../types/RegistrationTypes";
+import ReviewInformation from "./ReviewInformation";
```

Comment:
This is just used to setup the component and to have the linter be mad about it being dead code, I assume @twilkhoo's ticket will build out this component